src/styles/utils/_mixins.scss

```
@@ -1,6 +1,88 @@
+@import "variables";
+
 @mixin hover($_property, $_toValue) {
```

If you want to keep this mixin my suggestion is to name it differently so it is not used in unproper way:
```suggestion
@mixin transitionOnHover($_property, $_toValue) {
```

---

But I see this mixing is causing a lot of troubles so I would advise to remove it and add hover effect directly with proper transition on original element when it is possible :)

---

I deleted this mixin.